app/controllers/groups_controller.rb

```diff
@@ -0,0 +1,19 @@
+class GroupsController < ApplicationController
+    include GroupsHelper
+
+    def show
+        @group = Group.find(params[:id])
+        @projects = Project.all
+    end
+    
+    def update
+        @group = Group.find(params[:id])
+        @projects = Project.all
+        @group[:first_choice] = params[:first_choice]
+        @group[:second_choice] = params[:second_choice]
+        @group[:third_choice] = params[:third_choice]
```

Comment:
Not sure. One thing I'll note is that @group.first_choice = params[:first_choice] does *not* work. You have to use the brackets. I dunno why - maybe someone can explain this to me.